pkgs/development/python-modules/cfg_diag/default.nix

```diff
@@ -0,0 +1,44 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, pytest
+, python
+, setuptools
+}:
+
+let
+  cfg_diag = buildPythonPackage rec {
+    pname = "cfg_diag";
+    version = "0.4.0";
+    format = "pyproject";
+
+    src = fetchFromGitHub {
+      owner = "storpool";
+      repo = "python-cfg_diag";
+      rev = "release/${version}";
+      sha256 = "y9g/0JzbeF1gab/HaTfKj1AtAdcY6LO1aeEn+A2Ms/s=";
+    };
+
+    meta = with lib; {
+      homepage = "https://github.com/storpool/python-cfg_diag";
+      description = "A common configuration-storage class with a .diag() method";
+      license = licenses.bsd2;
+      maintainers = with maintainers; [ ppentchev ];
+    };
+
+    propagatedBuildInputs = [
+      setuptools
+    ];
+
+    checkInputs = [
+      pytest
+    ];
+
+    checkPhase = ''
+      runHook preCheck
+      PYTHONPATH="''${PYTHONPATH+''$PYTHONPATH:}$(pwd)/src" ${python.interpreter} -m pytest -v unit_tests
+      runHook postCheck
+    '';
+  };
+in
+cfg_diag
```

Comment:
What is the purpose of this? I don't see any self-references here.

Reply:
This was mainly a leftover of my confusion about checking the installed package vs running the unit tests on the source; see below. Thanks for catching this, fixed by removing it.